app/views/external_pledges/create.js.erb

```diff
@@ -1 +1 @@
-// rerender #existing-external-pledges
\ No newline at end of file
+$('#external_pledge_amount').val('');
```

Comment:
For some reason I can't get the external_pledge_amount to clear, no idea why. If you got an extra eye, go for it!